test/fixtures/servers/http.js

```diff
@@ -48,4 +48,26 @@ httpServer.on('/custom-reason', function (req, res) {
     res.end();
 });
 
+httpServer.on('/file-upload', function (req, res) {
+    if (req.method === 'POST') {
+        var body = '';
+
+        req.on('data', function (data) {
+            body += data;
+        });
+
+        req.on('end', function () {
+            try {
+                res.setHeader('content-length', String(body.length));
```

Comment:
`content-length` response header signifies the length of the response body but here response is empty.